### DISCUSSION TITLE
Sending text and an image in one photo

### DISCUSSION THREAD

**Discussion by ognjensold** [eternnoir/pyTelegramBotAPI#1677] [General]
Hi!

I want to know how to send a message with an image above.

Example provided below
![image](https://user-images.githubusercontent.com/98059817/185791881-250a8d41-b9ad-45bc-97e1-3c7290471ed7.png)

**Comment by ognjensold** [comment]
Hi!

Found a solution, I do not know how to delete this discussion....

**Reply by coder2020official** [reply]
Hello, sorry for late response. You can set it as an answer

**Comment by Badiboy** [comment]
I want to know how to send a message with an image above.

Upload image to some host with direct link (may use telegra.ph).
Add link to any word or letter. Image will appear on the top.
